docs/manuals/reference_rest/index.md

```diff
@@ -0,0 +1,6 @@
+# Taipy's REST API
```

Comment:
This file is related to ref_man and not user_man. I would not put this file under `manuals` folder, but I would create another folder `reference`. 
The index.md file path would be `docs/reference/rest/index.md`.

Does it make sense to you?

Reply:
It does. This should be the 'reference_rest' directory.
Cannot use 'reference', which is 100 % generated...
Thanks!

Reply:
Oh. I was going to fix this.... but I don't need to.
We do agree: this file is specific to the Reference REST API Manual. And in my opinion, located at the right place.
Do you agree with that?